tests/testthat/test-read-csv.R

```diff
@@ -0,0 +1,37 @@
+context("read_csv parameter testing")
+
+#Formatting for column imputation, col_name detection and NA detection.
+test_data <- read_csv(file.path(getwd(),"/basic-df.csv"), col_types = NULL, col_names = TRUE)
+
+test_that("read_csv automatic column type imputation works",{
+  expect_that(unname(unlist(lapply(test_data,class))), equals(c("logical","integer","numeric","character")))
+})
+
+test_that("read_csv's col_names = TRUE actually picks up col_names",{
+  expect_that(names(test_data), equals(c("a","b","c","d")))
+})
+
+test_that("read_csv detects NA values",{
+  expect_that(sum(is.na(test_data$d)), equals(1))
+})
+
+test_that("read_csv's 'NA' option genuinely changes the NA values",{
+  test_data <- read_csv(file.path(getwd(),"/basic-df.csv"), na = "turnip")
```

Comment:
Just `"basic-df.csv"` should be fine - tests are run with wd set to `tests/testthat`